OHHTTPStubs/OHHTTPStubs.xcodeproj/project.pbxproj

```diff
@@ -1329,7 +1331,7 @@
 				MODULEMAP_FILE = "$(SRCROOT)/Supporting Files/module.modulemap";
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
-				SWIFT_VERSION = 3.0;
+				SWIFT_VERSION = 4.0;
```

Comment:
I think that we must not bump the swift version here because we are still supporting Swift 3.x and the CI still run tests using Xcode 8.3.

Reply:
The hole problem is that I can not use the Swift 3 version in carthage because Xcode fails if I include a framework build with Swift 3

Reply:
I understand your problem but this is something we cannot do without considering the fact that we still support Swift 3 and that the CI needs to be able to check that for us.
AFAIK, the support of multiple Swift versions is not possible with Carthage, we've even opened an [issue](https://github.com/Carthage/Carthage/issues/1445) about this that was not resolved.
See [this section](https://github.com/AliSoftware/OHHTTPStubs#using-the-right-swift-version-for-your-project) of the README for our proposed workarounds for Carthage.

Reply:
> AFAIK, the support of multiple Swift versions is not possible with Carthage, we've even opened an issue about this that was not resolved.

Xcode 9 and above support Swift 3.x and Swift 4 for each-target basis, so this should not be an issue nowadays.

Reply:
I'm fully aware that support Swift 3 is support by Xcode and would make this project work on the most version of Xcode. 
We just have an automated build system that required some modules in Swift 4 and it did not want to build with a Swift 3 module in place.